src/server.ts

```diff
@@ -510,10 +518,11 @@ export interface ServerDetails  {
 	mgmtIpNetmask: string;
 	offlineReason: string;
 	physLocation: string;
-	profile: string;
+	profile: string | null;
 	/** @deprecated This has been removed from the latest API version. */
-	profileDesc: string;
+	profileDesc: string | null;
```

Comment:
these two properties have been removed, and will never be either `string` or `null`